ddl/owner_manager.go

```diff
@@ -112,6 +114,17 @@ func (m *ownerManager) SetBgOwner(isOwner bool) {
 // ManagerSessionTTL is the etcd session's TTL in seconds. It's exported for testing.
 var ManagerSessionTTL = 60
 
+// setManagerSessionTTL sets the ManagerSessionTTL value, it's used for testing.
+func setManagerSessionTTL() error {
```

Comment:
This function has an error message that needs to be returned, so the current approach might be better.